packages/react-components/react-theme/src/global/typographyStyles.ts

```diff
@@ -26,7 +26,7 @@ export const typographyStyles: TypographyStyles = {
   body2: {
     fontFamily: tokens.fontFamilyBase,
     fontSize: tokens.fontSizeBase400,
-    fontWeight: tokens.fontWeightSemibold,
+    fontWeight: tokens.fontWeightRegular,
```

Comment:
This seems right to me and I'm guessing you already talked with design, I just wanted to double check that this was the desired weight for other body2 uses, e.g. Miro's PR for body2 (#24378) references adding a react-text Body2 component, though it looks like that hasn't happened yet.

Reply:
(as mentioned in chat) Yes.  This is a code bug.  Probably a cut-and-paste bug as the figma always said regular font weight.  Yvonne and Daisy agree.